libraries/plugins/apis/database_api/include/steem/plugins/database_api/database_api_args.hpp

```diff
@@ -600,6 +595,15 @@ struct get_smt_next_identifier_return
 };
 #endif
 
+#define DEFINE_API_ARGS( api_name, arg_type, return_type )  \
+typedef arg_type api_name ## _args;                         \
+typedef return_type api_name ## _return;
+    
+/*               API,                                    args,                return */
+DEFINE_API_ARGS( find_transaction,                      vector< variant >,    find_transaction  )
```

Comment:
Do not use `vector<variant>` as argument type. Use a struct. Follow example of other `database_api` methods, not `condenser_api`.